### ISSUE TITLE
Windows compatible?

### ISSUE THREAD

**Issue by Darthagnon** [pojntfx/multiplex#9]
Was just wondering if this awesome project was Windows-compatible? GoLang stuff usually is, but I saw references to GTK, which is a little more complicated on Windows.

**Comment by pojntfx** [comment]
Yup, it is! I made sure that it spawns MPV etc. correctly, and tested it on Windows 11. There used to be a complete CI/CD setup I've set up for Windows, but I decided to drop it before release since I can't really test it properly: https://github.com/pojntfx/multiplex/blob/b2e0f2eb5ae8cd97493c7075fcb793da8fb93f46/Hydrunfile#L54-L99

**Comment by pojntfx** [comment]
I'm def. open to building it on Windows again, but for that to be viable I would need to be able to build and test the Windows build from Linux for it to be maintainable.

**Comment by pojntfx** [comment]
While the web sharing idea would be optimal, implementing it is a bit challenging due to browser limitations and legal concerns. See: https://github.com/pojntfx/multiplex/issues/5

**Comment by gmankab** [comment]
@pojntfx, can you publish windows build on releases tab please?

**Comment by pojntfx** [comment]
> @pojntfx, can you publish windows build on releases tab please?

I'm actively looking into it now that I got WINE on MSYS2 to work again, but I'm still not sure if I can genuinely support the platform considering that I don't really have a machine to do the testing for it on. See https://github.com/pojntfx/multiplex/issues/9#issuecomment-1712958270

**Comment by gmankab** [comment]
you can just publish it and community will test it :)

**Comment by pojntfx** [comment]
That could be a way of doing it, but publishing a release without being able to test it on my end will eventually lead to a lot of issues that I can't really account for.

**Comment by gmankab** [comment]
:(

**Comment by pojntfx** [comment]
Not much yet sadly! Lots of refactoring in process atm for the core components behind the project, mostly https://github.com/pojntfx/panrpc and https://github.com/pojntfx/weron. I'm maintaining the project in the meantime - I'm also planning on switching to a proper embedded player once that's all ready - but it will still be a bit longer.

In the meantime, if you want to build on macOS or Windows, you can try building from source: https://github.com/pojntfx/multiplex?tab=readme-ov-file#contributing
